### ISSUE TITLE
Species position in motif_census_triplets

### ISSUE THREAD

**Issue by david-beauchesne** [KevCaz/motifcensus#9]
There seems to be an issue with the species positions identified with `motif_census_triplets()`. When I run the analysis, the position numbers are not as identified in the function documentation for the food chains, omnivory, apparent and competitive exploitation. The positions are:
```1  2  3  4  5  9 10 11 12 13```

```{r}
uid <- unique(table64$name_uni)[-1]
unique
apply(table64[, c('pos_i','pos_j','pos_k')], 2, unique)
unique(paste(table64[, c('pos_i','pos_j','pos_k')]))

there are a limited number of positions (_i.e._ `pos_i`, `pos_j`, `pos_k`) that are identified with the numbers 11, 12 and 13. 

Those appear to be:
```{r}
11: omnivory middle
12: omnivory bottom
13: omnivory top 
```
Are those position results normal and how do they differ from the position 8, 9 and 10?

I can provide code for it if you want, I just have no reproducible example at the moment.

**Comment by david-beauchesne** [comment]
Here is the current classification: 

```
1: exploitative competition bottom
2: exploitative competition top
3: linear chains bottom
4: linear chains middle
5: linear chains top
9: apparent competition bottom
10: apparent competition top
11: omnivory bottom
12: omnivory middle
13: omnivory top
```

**Comment by KevCaz** [comment]
Yes, there is a missing piece in the documentation.... So far we actually have 2 different sets of ids: the one you've mentioned and the one for networks with bidirectional interactions that includes 64 possibilities and 30 unique positions. I used the latter nomenclature. And currently there is no conversion table that would identify the 13 unidirectional positions among the 30 bidirectional ones. This needs to be done.
